insights/parsers/messages.py

```diff
@@ -86,3 +86,23 @@ def get(self, s):
                     msg_info['procname'] = info_splits[4]
                 r.append(msg_info)
         return r
+
+    def get_after(self, timestamp, lines=None):
+        """
+        Get a list of lines after the given time stamp.
+
+        Parameters:
+            timestamp(datetime.datetime): log lines after this time are
+                returned.
+            lines(list): the list of log lines to search (e.g. from a get).
+                If not supplied, all available lines are searched.
+
+        Returns:
+            (list): The list of log lines with time stamps after the given
+                date and time.
```

Comment:
Yeah - I keep on thinking Sphinx needs that when it doesn't.  Will fix.

In general what I should have done is to get the whole patch completed and okayed, then submit the PR for the 1.x code.  Much easier that way.